density/density.py

```diff
@@ -493,5 +493,27 @@ def map():
     # Render template has an SVG image whose colors are changed by % full
     return render_template('map.html', locations=locations)
 
+@app.route('/upload', methods = ['POST'])
+def upload():
+    """ Accept POST requests from CUIT to add new data to the server """
+
+    # This is stored in local settings and is the way we verify uploads.
+    secret_key = request.args['key']
+    if secret_key != app.config['UPLOAD_KEY']:
+        return 'Please include a valid API key.', 401
```

Comment:
Gonna keep it as a key, I think it should be easier for CUIT to do it as a URL param.